### PR TITLE
remove aad support

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_client.py [hunk 1]
```diff
@@ -88,25 +88,11 @@ def __init__(self, endpoint, credential, **kwargs):
         if not credential:
             raise ValueError("Missing credential")
 
-        self._config = AzureCognitiveServiceMetricsAdvisorRESTAPIOpenAPIV2Configuration(endpoint=endpoint, **kwargs)
-        self._endpoint = endpoint
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_client.py:92]: This is needed for the __repr__ you have defined

File: sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_administration_client.py [hunk 2]
```diff
@@ -177,13 +177,21 @@ def __init__(self, endpoint, credential, **kwargs):
...
         self._client = _Client(
             endpoint=endpoint,
             sdk_moniker=SDK_MONIKER,
             authentication_policy=MetricsAdvisorKeyCredentialPolicy(credential),
             **kwargs
         )
 
+    def __repr__(self):
+        # type: () -> str
+        return "<MetricsAdvisorKeyCredential [endpoint={}]>".format(
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_administration_client.py:191]:
```suggestion
        return "<MetricsAdvisorAdministrationClient[endpoint={}]>".format(
```

**Comment by kristapratico** [comment]
Can we update the async clients as well?

**Review by xiangyan99 — COMMENTED** [review]
What should I say?

We are so lucky for having you. :)

**Review by kristapratico — APPROVED** [review]

**Pull request merged**